mpact/models.py

```diff
@@ -30,21 +30,33 @@ def __str__(self):
 
 class Bot(models.Model):
     chat = models.OneToOneField(Chat, on_delete=models.CASCADE)
```

Comment:
How come this is OneToOne? Shouldn't it be ManyToMany? (Although, for now, we would just have one bot.)

Reply:
yeah, I forgot to change it. Actually, it should be one to many as there can't be multiple bots in a single group.